Client/mods/deathmatch/logic/CClientMarker.cpp

```diff
@@ -412,26 +412,42 @@ void CClientMarker::StreamOut()
 
 void CClientMarker::Callback_OnCollision(CClientColShape& Shape, CClientEntity& Entity)
 {
-    if (IS_PLAYER(&Entity))
-    {
-        // Call the marker hit event
-        CLuaArguments Arguments;
-        Arguments.PushElement(&Entity);                                              // player that hit it
-        Arguments.PushBoolean((GetDimension() == Entity.GetDimension()));            // matching dimension?
-        CallEvent("onClientMarkerHit", Arguments, true);
-    }
+    if (GetInterior() != Entity.GetInterior())
+        return;
+
+    // Call the marker hit event
+    CLuaArguments Arguments;
+    Arguments.PushElement(&Entity);                                            // Hit element
+    Arguments.PushBoolean(GetDimension() == Entity.GetDimension());            // Matching dimension?
+    CallEvent("onClientMarkerHit", Arguments, true);
+
+    if (!IS_PLAYER(&Entity))
+        return;
+
+    CLuaArguments Arguments2;
+    Arguments2.PushElement(this);                                               // marker
```

Comment:
Can you change the case to camelCase?
`Arguments` -> `arguments`;
`Arguments2` -> `arguments2`; etc.

Reply:
The old code was like this, but I will modify it for further maintainability